### PR TITLE
[#162048444] Replace old Markdown component with the new MarkdownViewer

### PR DESCRIPTION
**Pull request by fpersico**

### CODE REVIEW

**Comment by digitalcitizenship** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true">Warnings</th>
    </tr>
  </thead>
  <tbody><tr>
      <td>:warning:</td>
      <td>

  Please include a description of your PR changes.
  </td>
    </tr>
  </tbody>
</table>



## Affected stories

  * 🌟 [#162048444](https://www.pivotaltracker.com/story/show/162048444): Il componente Markdown va rivisto/riscritto

<p align="right">
  Generated by :no_entry_sign: <a href="http://github.com/danger/danger-js/">dangerJS</a>
</p>

File: ts/IdentificationModal.tsx [hunk 1]
```diff
@@ -46,9 +46,7 @@ type State = {
 const contextualHelp = {
   title: I18n.t("pin_login.unlock_screen.help.title"),
   body: () => (
-    <Markdown lazyOptions={{ lazy: true }}>
-      {I18n.t("pin_login.unlock_screen.help.content")}
-    </Markdown>
+    <MarkdownViewer markdown={I18n.t("pin_login.unlock_screen.help.content")} />
```
Comment (cloudify) [inline ts/IdentificationModal.tsx]: couple of points here:

* let's keep the name of the component as before (i.e. `Markdown`) the `Viewer` suffix doesn't seem to add anything
* the children based syntax feels much more intuitive than passing the markdown via a prop - any reason to not keep it?

File: ts/components/wallet/PaymentMethodsList.tsx [hunk 2]
```diff
@@ -127,5 +127,5 @@ class PaymentMethodsList extends React.Component<Props, never> {
 export default withContextualHelp(
   PaymentMethodsList,
   I18n.t("wallet.whyAFee.title"),
-  () => <Markdown>{I18n.t("wallet.whyAFee.text")}</Markdown>
+  () => <MarkdownViewer markdown={I18n.t("wallet.whyAFee.text")} />
```
Comment (cloudify) [inline ts/components/wallet/PaymentMethodsList.tsx]: let's keep the old syntax :)

File: ts/components/messages/MessageMarkdown.tsx [hunk 3]
```diff
@@ -0,0 +1,18 @@
+import { Omit } from "italia-ts-commons/lib/types";
+import React from "react";
+
+import Markdown, { MarkdownProps } from "../Markdown";
+
+type Props = Omit<MarkdownProps, "cssStyle">;
+
+const MESSAGE_CSS_STYLE = `
+body {
+  font-size: 16px;
```
Comment (cloudify) [inline ts/components/messages/MessageMarkdown.tsx]: any chance we can derive the font size from the NativeBase theme? (`variables`)

File: ts/components/screens/MarkdownScreenComponent.tsx [hunk 4]
```diff
@@ -4,8 +4,8 @@ import { InteractionManager } from "react-native";
 
 import themeVariables from "../../theme/variables";
 import { ComponentProps } from "../../types/react";
+import Markdown from "../Markdown";
 import ActivityIndicator from "../ui/ActivityIndicator";
-import Markdown from "../ui/Markdown";
```
Comment (cloudify) [inline ts/components/screens/MarkdownScreenComponent.tsx]: why the need to move the component out of the `ui` directory?

File: ts/screens/preferences/ServiceDetailsScreen.tsx [hunk 5]
```diff
@@ -319,11 +313,7 @@ class ServiceDetailsScreen extends React.Component<Props, State> {
               </Col>
             </Row>
             <View spacer={true} large={true} />
-            {description && (
-              <Markdown lazyOptions={{ lazy: true, animated: true }}>
-                {description}
-              </Markdown>
-            )}
+            {description && <Markdown>{description}</Markdown>}
```
Comment (cloudify) [inline ts/screens/preferences/ServiceDetailsScreen.tsx]: we probably need to add the `animated` props as well, it used to animate the layout of the Markdown component in the service detail screen, as there's some content after the markdown - can you please check that with the current component the layout doesn't change abruptly ?

**Comment by cloudify** [comment]
Excellent job, thanks @fpersico !

**Pull request merged**